src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=<enter operator name here>
+publisher=<enter publisher name here>
+wat.warcinfo.description=<enter warc info description here>
```

Comment:
I've corrected the code and the properties file as you suggested, please
have a look.